typespec-tests/src/test/java/com/type/property/nullable/CollectionsStringClientTest.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.type.property.nullable;
+
+import com.type.property.nullable.models.CollectionsStringProperty;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+
+public class CollectionsStringClientTest {
+
+    private final CollectionsStringClient client = new NullableClientBuilder().buildCollectionsStringClient();
+
+    @Test
+    public void patchNonNullWithResponse() {
```

Comment:
remove `WithResponse` suffix as you are testing convenient methods? Same as below.

Reply:
Fixed in the new version.